app/src/lib/stores/app-store.ts

```diff
@@ -2212,6 +2218,16 @@ export class AppStore extends TypedBaseStore<IAppState> {
         PullRequestSuggestedNextAction
       ) ?? defaultPullRequestSuggestedNextAction
 
+    // Users who have never saved this preference should get the default of true
+    if (getBoolean(underlineLinksKey) === undefined) {
+      setBoolean(underlineLinksKey, true)
+    }
+
+    // Always false if the feature flag is disabled.
+    this.underlineLinks = enableLinkUnderlines()
+      ? getBoolean(underlineLinksKey, underlineLinksDefault)
+      : false
+
```

Comment:
```suggestion
    // Always false if the feature flag is disabled.
    this.underlineLinks = enableLinkUnderlines()
      ? getBoolean(underlineLinksKey, underlineLinksDefault)
      : false

```


> getBoolean(underlineLinksKey, underlineLinksDefault)

Because of `underlineLinksDefault = true`, the above line will make it true for everyone, so no need to set it. We only explicitly set it if we foresee needing to maintain current behavior for exist users (have it set to false).